nisystemlink/clients/auth/models/_auth_models.py

```diff
@@ -0,0 +1,171 @@
+"""Models utilized for Auth in SystemLink."""
+
+from __future__ import annotations
+
+from datetime import datetime
+from enum import Enum
+from typing import Any, Dict, List, Optional
+
+from nisystemlink.clients.core._uplink._json_model import JsonModel
+from pydantic import Field
+
+
+class AuthStatement(JsonModel):
+    """Auth Statement information."""
+
+    actions: Optional[List[str]] = None
+    """A list of actions the user is allowed to perform."""
+    resource: Optional[List[str]] = None
+    """A list of resources the user is allowed to access."""
+    workspace: Optional[str] = Field(None, example="workspace-id")
+    """The workspace the user is allowed to access."""
+
+
+class AuthPolicy(JsonModel):
+    """Auth Policy information."""
+
+    statements: Optional[List[AuthStatement]] = None
+    """A list of statements defining the actions the user can perform on a resource in a workspace.
+    """
+
+
+class Statement(JsonModel):
+    """Statement information."""
+
+    actions: Optional[List[str]] = None
+    """A list of actions the user is allowed to perform."""
+    resource: Optional[List[str]] = None
+    """A list of resources the user is allowed to access."""
+    workspace: Optional[str] = Field(None, example="workspace-id")
+    """The workspace the user is allowed to access."""
+    description: Optional[str] = None
+    """A description for this statement."""
```

Comment:
Simplify the definition
```suggestion
class Statement(AuthStatement):
    """Statement information."""

    description: Optional[str] = None
    """A description for this statement."""
```